x-pack/plugins/lens/public/indexpattern_datasource/field_item.scss

```diff
@@ -50,23 +50,6 @@
   color: $euiColorDarkShade;
 }
 
-.lnsFieldItem__topValue {
-  margin-bottom: $euiSizeS;
-
-  &:last-of-type {
-    margin-bottom: 0;
-  }
-}
-
-.lnsFieldItem__topValueProgress {
-  background-color: $euiColorLightestShade;
-
-  // sass-lint:disable-block no-vendor-prefixes
-  &::-webkit-progress-bar {
-    background-color: $euiColorLightestShade;
-  }
-}
-
```

Comment:
Styles were moved to src/plugins/unified_field_list/public/components/field_stats/field_stats.tsx file